test/e2e/specs/editor/blocks/separator.spec.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Separator', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'can be created by three dashes and enter', async ( {
+		editor,
+		page,
+	} ) => {
+		await page.locator( 'role=button[name="Add default block"i]' ).click();
```

Comment:
We can just click on the selector.

```suggestion
		await page.click( 'role=button[name="Add default block"i]' );
```